modules/packages/TOML.chpl

```diff
@@ -453,138 +453,26 @@ module TomlParser {
   }
 
 
-@chpldoc.nodoc
-// Enum for Toml class field: tag
- enum fieldtag {
-   fieldBool,
-   fieldInt,
-   fieldArr,
-   fieldToml,
-   fieldReal,
-   fieldString,
-   fieldEmpty,
-   fieldDate,
-   fieldTime,
-   fieldDateTime }
- private use fieldtag;
-
- @chpldoc.nodoc
- operator Toml.=(ref t: shared Toml, s: string) {
-   compilerWarning("= overloads for Toml are deprecated");
-   if t == nil {
-     t = new shared Toml(s);
-   } else {
-     t.tag = fieldString;
-     t.s = s;
-   }
- }
-
- @chpldoc.nodoc
- operator Toml.=(ref t: shared Toml, i: int) {
-   compilerWarning("= overloads for Toml are deprecated");
-   if t == nil {
-     t = new shared Toml(i);
-   } else {
-     t.tag = fieldInt;
-     t.i = i;
-   }
- }
-
- @chpldoc.nodoc
- operator Toml.=(ref t: shared Toml, b: bool) {
-   compilerWarning("= overloads for Toml are deprecated");
-   if t == nil {
-     t = new shared Toml(b);
-   } else {
-     t.tag = fieldBool;
-     t.boo = b;
-   }
- }
-
- @chpldoc.nodoc
- operator Toml.=(ref t: shared Toml, r: real) {
-   compilerWarning("= overloads for Toml are deprecated");
-   if t == nil {
-     t = new shared Toml(r);
-   } else {
-     t.tag = fieldReal;
-     t.re = r;
-   }
- }
-
- @chpldoc.nodoc
- operator Toml.=(ref t: shared Toml, ld: date) {
-   compilerWarning("= overloads for Toml are deprecated");
-   if t == nil {
-     t = new shared Toml(ld);
-   } else {
-     t.tag = fieldDate;
-     t.ld = ld;
-   }
- }
-
- @chpldoc.nodoc
- operator Toml.=(ref t: shared Toml, ti: time) {
-   compilerWarning("= overloads for Toml are deprecated");
-   if t == nil {
-     t = new shared Toml(ti);
-   } else {
-     t.tag = fieldTime;
-     t.ti = ti;
-   }
- }
-
- @chpldoc.nodoc
- operator Toml.=(ref t: shared Toml, dt: dateTime) {
-   compilerWarning("= overloads for Toml are deprecated");
-   if t == nil {
-     t = new shared Toml(dt);
-   } else {
-     t.tag = fieldDateTime;
-     t.dt = dt;
-   }
- }
-
- @chpldoc.nodoc
- operator Toml.=(ref t: shared Toml,
-                 A: [?D] shared Toml) where D.isAssociative() {
-   compilerWarning("= overloads for Toml are deprecated");
-   setupToml(t, A);
- }
- @chpldoc.nodoc
- proc setupToml(ref t: shared Toml, A: [?D] shared Toml) where D.isAssociative() {
-   if t == nil {
-     t = new shared Toml(A);
-   } else {
-     t.tag = fieldToml;
-     t.D = D;
-     t.A = A;
-   }
- }
-
- @chpldoc.nodoc
- proc setupToml(ref t: shared Toml, arr: [?dom] shared Toml) where !dom.isAssociative(){
```

Comment:
I'm mildly concerned about this. This is non-`private`, and not marked deprecated. A user who happened to rely on this wouldn't know not to use it or that it's going away.

That said, the whole package module is unstable, so I'd err on the side of removing this. Just thought I'd note it.

Reply:
Its marked private by the nature of it being `@chpldoc.nodoc`, so users who rely on it are relying on internal implementation details